core/arch/arm/plat-imx/conf.mk

```diff
@@ -178,6 +182,13 @@ else
 $(error Unsupported PLATFORM_FLAVOR "$(PLATFORM_FLAVOR)")
 endif
 
+ifneq (,$(filter $(PLATFORM_FLAVOR),mx6ulevk))
+CFG_IMX_I2C ?= n
+ifeq ($(CFG_IMX_I2C), y)
+CFG_IMX_I2C_CLK_RATE ?= 24000000
+endif
+endif
+
```

Comment:
Why not just add:
```
CFG_IMX_I2C_CLK_RATE ?= 24000000
```
...or perhaps:
```
CFG_IMX_I2C ?= n
CFG_IMX_I2C_CLK_RATE ?= 24000000
```
...near the end of the file, for all boards? It doesn't matter much if variables are defined but not used.
I let other iMX guys comment based on their preference ;-)

Reply:
yes I like your suggestion better...shall I wait for comments or just go with your proposal?